modules/puppet/manifests/puppetserver/config.pp

```diff
@@ -1,6 +1,8 @@
 # FIXME: This class needs better documentation as per https://docs.puppetlabs.com/guides/style_guide.html#puppet-doc
 class puppet::puppetserver::config {
 
+  $java_args = '-Xms8g -Xmx8g -XX:MaxPermSize=1g -Dhttps.protocols=TLSv1.2'
```

Comment:
The `-Dhttps.protocols=TLSv1.2` was added to the config on AWS Staging temporarily by me earlier and didn't have the desired effect, so this should be taken back out.